### PR TITLE
Aftershock update: Bionics and atomic gadgets

### PR DESCRIPTION
**Pull request by Ilysen**
#### Summary
```SUMMARY: Mods "Aftershock now has new content, including bionics, vehicle parts, and tools."```

#### Describe the solution
Aftershock gets some new goodies.
* KitchenMaster cooking buddy, a combination of the FOODCO and cooking rig that has their effects.
* The MetalMaster forge buddy now has a kiln.
* Adds the Linguistic Coprocessor CBM, which increases reading speed by 15%.
* Adds the Dopamine Stimulators CBM, which grants a constant mood elevation at the cost of 5 bionic power per 200 turns.
* Adds the atomic reading light and atompot; the atomic reading light is self-explanatory, and the atompot functions as a mess kit with limitless charges. Find them in places you find regular reading lights.
* Removed some duplicate json.
* A bunch of misc. balance changes, and shocker brutes can now drop Aftershock CBMs.

This update involves C++; I've been told it's fine to implement code used solely by mods. I've left comments mentioning that the code is used by Aftershock.

### CODE REVIEW

**Comment by Photoloss** [comment]
Is the atomic reading light really necessary? I'd rather see the vanilla nightlight buffed to avoid item bloat if that small change in brightness (5 to 8) truly makes a significant difference. Plus we have the atomic lamp anyway.

**Comment by Ilysen** [comment]
Maybe not necessary, but I like it. Reading lights are strong because of how little space they take up; the atomic reading light shares this strength, but the vanilla one still takes up less space (the atomic one is bulkier.) It is more of a sidegrade than a straight upgrade; a reading light a UPS mod takes up less space overall, but for those who just want a fully infinite light instead of a pseudo-infinite one, the atomic one makes a nice thing to have.

**Comment by VileBeggar** [comment]
I love this mod and I'm happy to see more stuff come out of it, cheers!

**Comment by Ilysen** [comment]
Are there any other steps necessary for this PR?

**Comment by mlangsdorf** [comment]
Still getting linting errors:
Makefile:1007: recipe for target 'data/mods/Aftershock/player/afs_professions.json' failed

**Comment by Ilysen** [comment]
I linted through every single file, so surely it'll work now!

File: data/mods/Aftershock/maps/afs_lab_weapons_range.json [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Cataclysm: Dark Days Ahead JSON Web Linting Tool
```
Comment (AMurkin) [inline data/mods/Aftershock/maps/afs_lab_weapons_range.json]: Here is the text from the json formatter page.

Comment (Ilysen) [reply data/mods/Aftershock/maps/afs_lab_weapons_range.json]: GAH.

File: data/mods/Aftershock/maps/afs_lab_weapons_range.json [hunk 2]
```diff
@@ -68,3 +68,4 @@
     }
   }
 ]
+
```
Comment (AMurkin) [inline data/mods/Aftershock/maps/afs_lab_weapons_range.json]:
```suggestion
```

**Pull request merged**